Source/Orts.Formats.Msts/WagonFile.cs

```diff
@@ -38,6 +63,10 @@ public WagonFile(string filePath)
                         stf.ReadString();
                         stf.ParseBlock(new STFReader.TokenProcessor[] {
                             new STFReader.TokenProcessor("name", ()=>{ Name = stf.ReadStringBlock(null); }),
+                            new STFReader.TokenProcessor("type", ()=>{ WagonType = stf.ReadStringBlock(null); }),
+                            new STFReader.TokenProcessor("mass", ()=>{ MassKG = stf.ReadFloatBlock(STFReader.UNITS.Mass, null); }),
+                            new STFReader.TokenProcessor("size", ()=>{ LengthM = new CarSize( stf).CarLength; }),
```

Comment:
Wouldn't it make more sense to put the `CarSize` as a whole into a field?

Reply:
Yes. Done.